Repository/ServiceEntityRepository.php

```diff
@@ -28,6 +28,13 @@ public function __construct(ManagerRegistry $registry, $entityClass)
     {
         $manager = $registry->getManagerForClass($entityClass);
 
+        if (null === $manager) {
+            throw new \LogicException(sprintf(
```

Comment:
Can you add a use statement for `LogicException`?

Also, is it possible to add a unit test which covers this scenario?

Reply:
both done